app-admin/lsat/lsat-0.9.8.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
```

Comment:
I dont see eutils being used in here, can it be dropped?

Reply:
Done.